packages/cli-v3/src/commands/deploy.ts

```diff
@@ -1821,6 +1828,10 @@ export async function copyAdditionalFiles(
   tempDir: string
 ): Promise<AdditionalFilesReturn> {
   const additionalFiles = config.additionalFiles ?? [];
+  const extraCACerts = config.extraCACerts ?? '';
+  if (extraCACerts) {
+    additionalFiles.push(extraCACerts);
+  }
```

Comment:
I think we should just handle this in `resolveConfig()` as well. It's going to get confusing with the path resolution step otherwise where `config.extraCACerts !== buildOptions.extraCACerts`

Reply:
handled in resolveConfig
https://github.com/triggerdotdev/trigger.dev/blob/f606d3e072224e2496651e24bfe3e0bfdd30b642/packages/cli-v3/src/utilities/configFiles.ts#L246-L259

Reply:
Love a good regexp